docs/source/03_tutorial/02_setup.md

```diff
@@ -25,15 +25,13 @@ This plugins must be used in an existing kedro project. If you do not have a ked
 
 For this tutorial and if you do not have a real-world project, I strongly suggest that you accept to include the proposed example to make a demo of this plugin out of the box.
 
-## Update the template of your kedro project
-In order to use the ``kedro-mlflow`` plugin, you need to perform 2 actions:
-1. Create an ``mlflow.yml`` file for [configuring mlflow in a dedicated file](../05_python_objects/05_Configuration.md).
-2. Update the ``src/PYTHON_PACKAGE/run.py`` to add the [necessary hooks](../05_python_objects/02_Hooks.md) to the project context. The ``MlflowPipelineHook`` manages the configuration and registers the PipelineML, while the ``MlflowNodeHook`` autolog the parameters.
+## Activate `kedro-mlflow` in your kedro project
+In order to use the ``kedro-mlflow`` plugin, you need to set up the his configuration and declare his hooks. those 2 actions are detailled in the follow.
```

Comment:
**its** configuration and **its** hooks. **T**hose 2 actions are detailled in the **following paragraph**